break문은 반복문 단 하나를 빠져나올때 쓰는 것

이중반복문 나오는법 1
https://redmuffler.tistory.com/446





이중반복문을 탈출하기 위해 위 사이트에서 나오는 방법과 같이
status = False  #변수를 설정해놓고
for _ in range(5):

   for _ in range(5):
      if a == b:
         print(result, '정답 출력')
         status = True
         break  #for문 하나 탈출

   if status == True:
      break  #바깥 for문도 탈출
#END
이 방식으로 해왔다. ok 이중반복문 나올땐 그냥 하던대로 그렇게 해라!

다만, 이중반복문이 아니라 삼중, 사중..반복문이 될때도 status변수 상태를 계속 체크하며 모든 반복문을 빠져나올것인가?
저 다중반복문 자체를 def func(): ~  해서 함수 안에 구현해놨다면, 그냥 return 해버리고 메서드 자체를 종료해버리면 된다.
하지만 함수 안에 구현해놓은 것이 아닌 메인 자체에 구현해놨다면 return을 쓸 수 없다.
그래서 그때는 원하는 최종 정답을 출력해버리고 exit()를 써서 프로그램을 종료해버리자. -> 이렇게 제출해도 백준 플랫폼에서는 '맞았습니다' 처리됨
exit()를 해버려 끝내버려도 크게 문제 없다 !  ===> '[구현]2578번 빙고' 문제를 풀다가 남겨놓는 메모. . . . 
